shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -609,14 +616,18 @@
   // This wrapper view masks the overlay view.
   overlay_view_wrapper.frame = CGRectMake(rect.x() / screenScale, rect.y() / screenScale,
                                           rect.width() / screenScale, rect.height() / screenScale);
-  // Set a unique view identifier, so the overlay wrapper can be identified in unit tests.
+  // Set a unique view identifier, so the overlay_view_wrapper can be identified in XCUITests.
   overlay_view_wrapper.accessibilityIdentifier =
       [NSString stringWithFormat:@"platform_view[%lld].overlay[%lld]", view_id, overlay_id];
 
   UIView* overlay_view = layer->overlay_view.get();
   // Set the size of the overlay view.
   // This size is equal to the device screen size.
-  overlay_view.frame = flutter_view_.get().bounds;
+  overlay_view.frame = [flutter_view_.get() convertRect:flutter_view_.get().bounds
```

Comment:
have you seen any side effects when the frame size is changed in every frame?  Try running https://github.com/flutter/flutter/tree/195a1cc413bde5f7c1d194204608c4bc20659124/dev/benchmarks/platform_views_layout with this change. That test has an animated container that should cause this value to change after each frame https://github.com/flutter/flutter/blob/195a1cc413bde5f7c1d194204608c4bc20659124/dev/benchmarks/platform_views_layout/lib/main.dart#L121

Reply:
Good point. I ran the benchmarks, there doesn't seem to be significant difference:

With my change:
```
  "average_frame_build_time_millis": 0.2156956521739131,
  "90th_percentile_frame_build_time_millis": 0.27,
  "99th_percentile_frame_build_time_millis": 0.316,
  "worst_frame_build_time_millis": 0.326,
  "missed_frame_build_budget_count": 0,
  "average_frame_rasterizer_time_millis": 4.833215053763439,
  "90th_percentile_frame_rasterizer_time_millis": 6.668,
  "99th_percentile_frame_rasterizer_time_millis": 7.84,
  "worst_frame_rasterizer_time_millis": 9.612,
  "missed_frame_rasterizer_budget_count": 0,
```

master:
```
  "average_frame_build_time_millis": 0.21664516129032269,
  "90th_percentile_frame_build_time_millis": 0.286,
  "99th_percentile_frame_build_time_millis": 0.305,
  "worst_frame_build_time_millis": 0.357,
  "missed_frame_build_budget_count": 0,
  "average_frame_rasterizer_time_millis": 4.918989130434783,
  "90th_percentile_frame_rasterizer_time_millis": 6.905,
  "99th_percentile_frame_rasterizer_time_millis": 8.33,
  "worst_frame_rasterizer_time_millis": 9.434,
  "missed_frame_rasterizer_budget_count": 0,
```

Reply:
did you see any issue in general? if you run the app, and play with it?  e.g. Could this change introduce jank or glitches?

Reply:
no, I didn't see this change introduce new janks or glitches.

Reply:
friendly ping @blasten 
Did you have other suggestions/comments?

Reply:
no more suggestions. Thanks for looking!